cmd/root.go

```diff
@@ -40,6 +40,9 @@ var (
 			// This runs before any subcommand, and cmd.CalledAs() returns the subcommand
 			// that was called. We want to use this for the weep method in the instance info.
 			metadata.SetWeepMethod(cmd.CalledAs())
+			// Add basic metadata to ALL future logs
+			metadata.AddMetadataToLogger()
```

Comment:
We could also pass `args` to this function so we can have more context on invocation.